docs/tutorials/microsoft365/getting-started-m365.md

```diff
@@ -0,0 +1,205 @@
+
+---
+
+## Step 3: Configure your M365 account
+
+
+### Create the Service Principal app
+
+A Service Principal is required to grant Prowler the necessary privileges.
+
+1. Access **Microsoft Entra ID**
+
+    ![Overview of Microsoft Entra ID](./img/microsoft-entra-id.png)
+
+2. Navigate to `Manage` > `App registrations`
+
+    ![App Registration nav](./img/app-registration-menu.png)
+
+3. Click `+ New registration`, complete the form, and click `Register`
+
+    ![New Registration](./img/new-registration.png)
+
+4. Go to `Certificates & secrets` > `+ New client secret`
+
+    ![Certificate & Secrets nav](./img/certificates-and-secrets.png)
+    ![New Client Secret](./img/new-client-secret.png)
+
+5. Fill in the required fields and click `Add`, then copy the generated value
+
+| Value | Description |
+|-------|-------------|
+| Client ID | Application ID |
+| Client Secret | AZURE_CLIENT_SECRET |
+| Tenant ID | Azure Tenant ID |
+
+---
+
+### Grant required API permissions
+
+Assign the following Microsoft Graph permissions:
+
+    - `Directory.Read.All`: Required for all services.
+    - `Policy.Read.All`: Required for all services.
+    - `User.Read` (IMPORTANT: this is set as **delegated**): Required for the sign-in.
+    - `Sites.Read.All`: Required for SharePoint service.
+    - `SharePointTenantSettings.Read.All`: Required for SharePoint service.
+
+1. Go to your App Registration > `API permissions`
+
+    ![API Permission Page](./img/api-permissions-page.png)
+
+2. Click `+ Add a permission` > `Microsoft Graph` > `Application permissions`
+
+    ![Add API Permission](./img/add-app-api-permission.png)
+    ![Microsoft Graph Detail](./img/microsoft-graph-detail.png)
+
+3. Search and select:
+
+    - `Directory.Read.All`
+    - `Policy.Read.All`
+    - `Sites.Read.All`
+    - `SharePointTenantSettings.Read.All`
+
+    ![Permission Screenshots](./img/directory-permission.png)
+
+4. Click `Add permissions`, then grant admin consent
+
+    ![Grant Admin Consent](./img/grant-admin-consent.png)
+
+5. Click `+ Add a permission` > `Microsoft Graph` > `Delegated permissions`
+
+    ![Add API Permission](./img/add-delegated-api-permission.png)
+    ![Microsoft Graph Detail](./img/microsoft-graph-detail.png)
+
+6. Search and select:
+
+    - `User.Read`
+
+    ![Permission Screenshots](./img/directory-permission.png)
+
+7. Click `Add permissions`, then grant admin consent
+
+    ![Grant Admin Consent](./img/grant-admin-consent.png)
+
+---
+
+### Assign required roles to your user
+
+Assign one of the following roles to your User:
+
+    - `Global Reader` (recommended): this allows you to read all roles needed.
+    - `Exchange Administrator` and `Teams Administrator`: user needs both roles but with this [roles](https://learn.microsoft.com/en-us/exchange/permissions-exo/permissions-exo#microsoft-365-permissions-in-exchange-online) you can access to the same information as a Global Reader (here you only read so that's why we recomend that role).
```

Comment:
```suggestion
- `Global Reader` (recommended): this allows you to read all roles needed.
- `Exchange Administrator` and `Teams Administrator`: user needs both roles but with this [roles](https://learn.microsoft.com/en-us/exchange/permissions-exo/permissions-exo#microsoft-365-permissions-in-exchange-online) you can access to the same information as a Global Reader (here you only read so that's why we recomend that role).
```